legend-engine-authentication-pure/src/main/resources/core_authentication/metamodel/metamodel.pure

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 Goldman Sachs
```

Comment:
I think we should discuss if we need new repo for authentication and it should not go into core directly